simulation_parameters/Constants.cs

```diff
@@ -1356,11 +1356,11 @@ public static class Constants
 
     public const float TIME_BEFORE_TUTORIAL_CAN_PAUSE = 0.01f;
 
-    public const float MICROBE_MOVEMENT_EXPLAIN_TUTORIAL_DELAY = 12.0f;
+    public const float MICROBE_MOVEMENT_EXPLAIN_TUTORIAL_DELAY = 3.5f;
     public const float MICROBE_MOVEMENT_EXPLAIN_TUTORIAL_DELAY_CONTROLLER = 1.0f;
-    public const float MICROBE_MOVEMENT_TUTORIAL_REQUIRE_DIRECTION_PRESS_TIME = 2.2f;
+    public const float MICROBE_MOVEMENT_TUTORIAL_REQUIRE_DIRECTION_PRESS_TIME = 1.6f;
```

Comment:
I think this should not be reduced until we have a popup for the player to pick their preferred movement mode.

Reply:
The problem here was that the buttons last a little longer than a lot of player's intuition on when a task should be completed, which I think is leading to players assuming they are supposed to be pressing combinations or something. To help with your concerns, I added a new change to both highlight the option to change the movement controls, and make the movement tutorial unclosable, in case someone needs more time (which I've seen happen once).